package.json

```diff
@@ -42,4 +42,4 @@
     "prettier": "^3.3.3",
     "progress": "^2.0.3"
   }
-}
+}
```

Comment:
Umm, why remove this end line?

Reply:
The new eslint-prettier (https://github.com/microsoft/webnn-developer-preview/pull/76) auto removed the last empty end line.

Reply:
Interesting. 🤔

Reply:
Curious, as that introduces 🚫 signs in GitHub CR's now, making it a questionable default.